packages/p2p/src/stores/my-profile-store.js

```diff
@@ -42,6 +42,8 @@ export default class MyProfileStore extends BaseStore {
     @observable should_show_add_payment_method_form = false;
     @observable should_show_edit_payment_method_form = false;
 
+    MODAL_TRANSITION_DURATION = 250;
```

Comment:
This constant is added in this PR https://github.com/binary-com/deriv-app/pull/6200 as part of universal constant value for all modal transition duration, adding it here for consistency